bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -220,9 +221,9 @@ export class PlotCanvasView extends BokehView
 
     if follow_enabled and has_bounds
       logger.warn('Follow enabled so bounds are unset.')
-      for xr in _.values(frame.x_ranges)
+      for xr in values(frame.x_ranges)
         xr.bounds = null
-      for yr in _.values(frame.y_ranges)
+      for yr in values(frame.y_ranges)
```

Comment:
if we implement `values` as a convenience we should definitely implement `keys` too I think